.travis.yml

```diff
@@ -19,7 +19,7 @@ install:
 
   - if [ ${TRAVIS_PYTHON_VERSION:0:1} == "2" ]; then conda install --yes mock; fi
   - pip install --no-deps numdifftools
-  - pip install git+https://github.com/Theano/Theano.git
+  - pip install git+https://github.com/Theano/Theano.git@01e2ff6acce2654c8bf0363ce915285f7173eb2f
```

Comment:
Yes, I reverted this on master as pymc broke twice with changes on the theano branch.